src/components/main-nav/nav-item.svelte

```diff
@@ -21,6 +21,10 @@
   .active {
     color: var(--black);
   }
+
+  :global(a) :global(sup) {
+    color: var(--brand-ripe);
```

Comment:
Here's the Tailwind way to do the same:

```suggestion
    @apply text-orange-800;
```